daemons/attrd/attrd_cib.c

```diff
@@ -71,7 +147,18 @@ attrd_cib_updated_cb(const char *event, xmlNode *msg)
     if (!cib__client_triggers_refresh(client_name)) {
         /* This change came from a source that ensured the CIB is consistent
          * with our attributes table, so we don't need to write anything out.
+         * If a removed attribute has been erased, we can forget it now.
          */
+        int format = 1;
+
+        crm_element_value_int(patchset, PCMK_XA_FORMAT, &format);
+        if (format != 2) {
+            crm_warn("Can't handle CIB diff format %d", format);
+            return;
+        }
+        // @WIP Do we have to break const ...
```

Comment:
Right, but `pcmk__xe_foreach_child()` requires a non-const argument. The comment means I hadn't decided whether it's worth the trouble to add a `pcmk__xe_foreach_child_const()` since is the only caller that needs it :)

Reply:
Leaning to not making a new function

Reply:
Yeah, I don't think it's worth it.  I'd also hate to establish the precedent of making _const versions of things.